packages/devextreme/js/__internal/scheduler/header/m_date_navigator.ts

```diff
@@ -39,8 +39,8 @@ export const getDateNavigator = (header, item) => {
 };
 
 const getPreviousButtonOptions = (header) => {
-  // @ts-expect-error text
-  const ariaMessage = messageLocalization.format('dxScheduler-navigationPrevious', getCurrentViewText(header));
+  const { currentView } = header;
+  const ariaMessage = messageLocalization.format(`dxScheduler-navigationPrevious${currentView}`);
```

Comment:
Maybe we better unify the names? Not 'previous day', 'previous month' etc, but 'previous page' or 'previous range'
While 'previous day' looks ok, 'previous agenda' or 'previous timeline month' is not user-friendly

Reply:
Since user can create custom views, I think I should just stick to 'Previous Page' or 'Previous Range' instead..

Reply:
Please don't get me wrong, users can create custom views, but they always have predefined type (like 'month', 'day' etc) 
https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#type

Custom views are only about setting some specific properties to these views. Like, user can create two month views with different grouping etc

Reply:
But settings can still introduce some troubles. Like [intervalCount](https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#intervalCount), which can make the view contain two weeks instead of one, and therefore the label should be also like "next two weeks" instead of "two week"

Reply:
In this case, I think the navigation labels can be as simple as 'Previous Page' 'Next Page'. As I see there might be a continuous conflict  if we use specific viewTypes with added logic.. What do you think? Please let me know if I should use the specific viewTypes with the required logic that can interpret intervalCount and other instances correctly..

Reply:
@Raushen @pomahtri , please review the latest commit. I haven't considered the intervalCount because if I will create it, we may not use message localization due to language differences. If I'm not to use messageLocalization, then the aria-label will only be available in english..